dev/wlp-jakartaee-transform/rules/jakarta-versions-ee10.properties

```diff
@@ -217,18 +217,18 @@ org.glassfish.jaxb.runtime.v2.runtime.unmarshaller=[3.0,4)
 org.glassfish.jaxb.runtime.v2.schemagen=[3.0,4)
 org.glassfish.jaxb.runtime.v2.schemagen.xmlschema=[3.0,4)
 org.glassfish.jaxb.runtime.v2.util=[3.0,4)
-org.hibernate.validator=[7.0,8)
```

Comment:
I'm not sure we actually need these. I believe HV8 switched from the javax to jakarta namespace, so I don't know that we need to run the transformer. We should try without these changes to confirm